.github/workflows/eluc.yml

```diff
@@ -0,0 +1,35 @@
+# This runs the unit tests for the ELUC use case
+
+name: ELUC Use Case Python Unit Tests
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Can we run it on all branch? Because we shouldn't be able to merge if pylint or the tests fail

Reply:
Currently runs whenever a PR goes into main